aliases/aliases.go

```diff
@@ -0,0 +1,127 @@
+package aliases
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	bolt "github.com/coreos/bbolt"
+	"github.com/gomeeseeks/meeseeks-box/db"
+)
+
+var aliasesBucketKey = []byte("aliases")
+
+// ErrAliasNotFound is returned when an alias can't be found
+var ErrAliasNotFound = fmt.Errorf("alias not found")
+
+// Alias represent an alias for a command
+type Alias struct {
+	Alias   string
+	Command string
+	Args    []string
+}
+
+// Add adds a new alias for a user ID
+func Add(userID, alias, command string) error {
+	return db.Update(func(tx *bolt.Tx) error {
+		bucket, err := getAliasesBucket(userID, tx)
+		if err != nil {
+			return err
+		}
+		cmdChunks := strings.Split(command, " ")
```

Comment:
humm.. this is a bit naive, I honestly think we need to either use the same parser, or just send the args as a slice instead of joining them back. The latter will be preferred as it will be less processing.

The reasoning behind this is that only splitting on spaces is, first of all not needed, second, it changes the meaning o the command if the arguments were actually being surrounded with quotes.

So, it's better to change the signature to `func Create(userId, alias, command string, args... string)` and then change the Alias to

```
Alias{
  Alias: alias,
  Command: command,
  Args: args,
}
```